### PR TITLE
User map: move 'last updated date' to userMapData.json 6317

### LINKED ISSUE #6317: User map: move 'last updated date' to userMapData.json
Currently the last updated date of the user map is in `src/main/webapp/usermap.jsp` while the user data is actually updated in `src/main/webapp/js/userMapData.json`.

Explore the possibility of moving the date into the json file so that only one file needs to be updated.

### PR DESCRIPTION
**Pull request by AnaghHegde**
Fixes #6317 #6322

### CODE REVIEW

**Comment by wkurniawan07** [comment]
@AnaghHegde 

```javascript
if (variable === ...) {
    // do something
} else {
    // do another thing
}
```
is preferred over
```javascript
if (variable !== ...) {
    // do something
} else {
    // do another thing
}
```
Also, may I suggest that you post your PR-related comments/inquiries on the PR itself instead of the issue thread?

**Comment by AnaghHegde** [comment]
@wkurniawan07 I am really sorry for not following the things and i won't repeat the same in future.Thanks for pointing out the mistake.It helped me a lot.

**Comment by AnaghHegde** [comment]
@wkurniawan07 Ready for review.

File: src/main/webapp/usermap.jsp [hunk 1]
```diff
@@ -15,7 +15,7 @@
     <h1 class="caption">Who is using TEAMMATES?</h1>
     <div id="contentHolder">
         <div id="container" style="position: relative; width: 800px; height: 500px; border: 1px solid #DEDEDE;"></div>
-        <p id="lastUpdate" class="lastUpdate">Last updated: 30 November 2016</p>
+        <p id="lastUpdate" class="lastUpdate">Last updated: <span id="lastUpdateDate" class="lastUpdate"></span></p>
```
Comment (wkurniawan07) [inline src/main/webapp/usermap.jsp]: For this line can you:
1. Remove `id="lastUpdate"` on the `p` tag? It does not seem to be used.
1. Remove `class="lastUpdate"` on the `class` tag? The effect due to the same class on the `p` tag is also spread here.

Comment (AnaghHegde) [inline src/main/webapp/usermap.jsp]: If i remove the class then the **Last Updated** phrase and the date will be in different style.But i will remove the id on the < p > tag.Is that ok?

**Comment by AnaghHegde** [comment]
@wkurniawan07 Ok noted. I will do those changes.

**Comment by AnaghHegde** [comment]
@wkurniawan07 one more thing to ask : Is it okay if i change the way in which data is store in the JSON ( not just adding headers) ?.

**Pull request closed without merging**